plugins/catalog-backend-module-github/src/module/catalogModuleGithubOrgEntityProvider.ts

```diff
@@ -0,0 +1,144 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { loggerToWinstonLogger } from '@backstage/backend-common';
+import {
+  coreServices,
+  createBackendModule,
+  createExtensionPoint,
+} from '@backstage/backend-plugin-api';
+import {
+  readTaskScheduleDefinitionFromConfig,
+  TaskScheduleDefinition,
+} from '@backstage/backend-tasks';
+import { Config } from '@backstage/config';
+import {
+  GithubMultiOrgEntityProvider,
+  TeamTransformer,
+  UserTransformer,
+} from '@backstage/plugin-catalog-backend-module-github';
+import { catalogProcessingExtensionPoint } from '@backstage/plugin-catalog-node/alpha';
+
+/**
+ * Interface for {@link githubOrgEntityProviderTransformsExtensionPoint}.
+ *
+ * @alpha
+ */
+export interface GithubOrgEntityProviderTransformsExtensionPoint {
+  /**
+   * Set a custom transformer for transforming from GitHub users to catalog
+   * entities.
+   */
+  setUserTransformer(transformer: UserTransformer): void;
+
+  /**
+   * Set a custom transformer for transforming from GitHub teams to catalog
+   * entities.
+   */
+  setTeamTransformer(transformer: TeamTransformer): void;
+}
+
+/**
+ * Extension point for runtime configuration of {@link catalogModuleGithubOrgEntityProvider}.
+ *
+ * @alpha
+ */
+export const githubOrgEntityProviderTransformsExtensionPoint =
+  createExtensionPoint<GithubOrgEntityProviderTransformsExtensionPoint>({
+    id: 'catalog.githubOrgEntityProvider',
+  });
+
+/**
+ * Registers the `GithubMultiOrgEntityProvider` with the catalog processing extension point.
+ *
+ * @alpha
+ */
+export const catalogModuleGithubOrgEntityProvider = createBackendModule({
+  pluginId: 'catalog',
+  moduleId: 'githubOrgEntityProvider',
+  register(env) {
+    let userTransformer: UserTransformer | undefined;
+    let teamTransformer: TeamTransformer | undefined;
+
+    env.registerExtensionPoint(
+      githubOrgEntityProviderTransformsExtensionPoint,
+      {
+        setUserTransformer(transformer) {
+          if (userTransformer) {
+            throw new Error('User transformer may only be set once');
+          }
+          userTransformer = transformer;
+        },
+        setTeamTransformer(transformer) {
+          if (teamTransformer) {
+            throw new Error('Team transformer may only be set once');
+          }
+          teamTransformer = transformer;
+        },
+      },
+    );
+
+    env.registerInit({
+      deps: {
+        catalog: catalogProcessingExtensionPoint,
+        config: coreServices.rootConfig,
+        logger: coreServices.logger,
+        scheduler: coreServices.scheduler,
+      },
+      async init({ catalog, config, logger, scheduler }) {
+        for (const definition of readDefinitionsFromConfig(config)) {
+          catalog.addEntityProvider(
+            GithubMultiOrgEntityProvider.fromConfig(config, {
```

Comment:
The usage of GithubMultiOrgEntityProvider doesn't seem to match the module name, but maybe that's intentional. Just pointing out in case it's an oversight.

Reply:
Yep indeed, good catch, it's intentional! There was really no reason to have two variants of the module, when all it boils down to, is that you add either one ore several config rows